coop_membership/models/res_config.py

```diff
@@ -34,6 +34,9 @@ class ResConfigSettings(models.TransientModel):
         string='FTOP Registration period', readonly=False)
     maximum_active_days = fields.Integer(
         related="company_id.maximum_active_days", readonly=False)
+    office_timing = fields.Text(
+        related='company_id.office_timing',
+        string='Office Timing', translate=True, readonly=False)
```

Comment:
```suggestion
    members_office_open_hours = fields.Text(
        related='company_id.members_office_open_hours',
        string="Members Office Open Hours", translate=True, readonly=False)
```

Would you please rename these fields as follows?